client/src/components/CommentForm/CommentForm.js

```diff
@@ -0,0 +1,256 @@
+    }
+    updateComment = (_id,author_id) => {
+        if(this.state.updateCommentText){
+            axios
+            .put(`/api/comment/update`, 
+            {
+                _id,
+                text: this.state.updateCommentText,
+                author_id
+            })
+            .then(() => {
+                this.getComments(this.props.entity_type,this.props.entity_id)
+                this.setState({updateCommentTextField: 0, updateCommentText: ''})
+            }).catch((err) => {
+                const errorText = err.response.data ? err.response.data.error.errors.text[0] : 'Something went wrong!'
+                this.showToast(errorText,'error');
+            }) 
+            }else {
+                this.setState({updateCommentTextField: 0});
+            }
+    }
+    showToast = (message, variant) => {
+        this.props.enqueueSnackbar(message, {
+            variant: variant ? variant : 'default',
+        });
+    };
+    render() {
+        const {comments} = this.state;
+        const renderComments = (
+        comments.map(comment => {
+            return (
+            <ExpansionPanel key={comment._id ? comment._id : Math.floor(Math.random() * 60) + 1} className="commentForm">
+                {this.state.user.id == comment.author_id ? 
+                <div>
+                { this.state.updateCommentTextField === 0 ?
+                    <Button
+                        className="editCommentBtn"
+                        onClick={() => this.updateTextField(comment._id)}
+                        >
+                        <Create/>
+                    </Button>
+                    :
+                    <div>
+                        {comment._id === this.state.updateCommentTextField ? 
+                        <Button
+                            className="editCommentBtn saveCommentBtn"
+                            onClick={() =>this.updateComment(comment._id, comment.author_id) }
+                            >
+                            <Done/>
+                        </Button>
+                        :
+                        <Button
+                            className="editCommentBtn"
+                            onClick={() => this.updateTextField(comment._id)}
+                            >
+                            <Create/>
+                        </Button>
+                        }
+                    </div>
+                }
+                    <Button
+                        className="deleteCommentBtn "
+                        onClick={() => this.deleteComment(comment._id, comment.author_id)}
+                    >
+                        <Delete/>
+                    </Button>
+                </div>
+                    : null}
+                <ExpansionPanelSummary className="event-page__faq-title" expandIcon={<ExpandMore />}  >
+                    <Avatar alt="avatar" src={comment.avatar} className="avatar  event-page__avatar" />
```

Comment:
You can add the avatar by default if the user has not installed the avatar: src={comment.avatar || defaultAvatar}
and import defaultAvatar from "../../assets/images/face.png";